### ISSUE TITLE
Labeling in popup fields

### ISSUE THREAD

**Issue by jukos** [qgis2web/qgis2web#833]
edit: working only with leaflet
Hello,
I have a wish:
I would edit the labels for several attributes with one click.
For example: 20 Attributes with inline label.

Why?
The reason is in QGIS.
Example:
I have many layers with many columns/attributes. I must often change the sort and the name of the columns. This I can only do with the qgis function "refactor fields". After edit all, qgis create a new layer. And for this new layer I must change the labels for the qgis2web popups again.
This is always unnecessary work. And I must change a layer sometimes often.
With a function/checkbox, so I can change all attributes to a specific label in one action, it would be very helpful.

Are there a possibility to integrate this function?
**"choose for all fields: no label or inline label or header label"**

Greetings, Jürgen

Edit:
@tomchadwin 
**Could I change the default setting in the plugin from "no label" to "inline label" ?** This would help me.
If yes, please explain me how I could do this. Thank you.

**Comment by riccardoklinger** [comment]
Hi Jürgen,

first of all: merry x-mas ;-) 
For your own purpose: take a look at line 808-810 of the file maindialog.py:
```
self.attrWidget.addItem("no label")
self.attrWidget.addItem("inline label")
self.attrWidget.addItem("header label")
```
If you switch order to:
```
                self.attrWidget.addItem("inline label")
                self.attrWidget.addItem("header label")
                self.attrWidget.addItem("no label")
```
It will work as desired. For a major rework and customizing I would consult @tomchadwin first...

**Comment by jukos** [comment]
It works. Thank you!

**Comment by tomchadwin** [comment]
I'll close this for now, but I agree that the default should probably be `inline label`.

**Comment by andreaordonselli** [comment]
Released version [3.17.3](https://github.com/qgis2web/qgis2web/releases/tag/v3.17.3) which includes a fix for this issue. The update will be available soon from the QGIS plugin repository. Thanks for the tip.
